.dir-locals.el

```diff
@@ -16,7 +16,7 @@
  ;; directories must be quoted, even if M-x add-dir-local-variable
  ;; adds them without quotes
  (c++-mode
-  (helm-make-build-dir . "build/debug/clang")
```

Comment:
nice. but separate change.

Reply:
this is already in its own change, you mean place this in a new PR altogether?

Reply:
yes, this emacs thing has nothing to do w/ `coproc router lifetime` which i also think needs to update the title for what the change is doing.